examples/adeo/plain.yaml

```diff
@@ -0,0 +1,326 @@
+    bindings:
+      kafka:
+        replicas: 3
+        partitions: 3
+        cleanup.policy: delete
+        retention.ms: 7 days
+    messages:
+      costingRequest:
+        $ref: "#/components/messages/costingRequestV1"
+
+
+
+
+
+  costingResponse:
+    address: "adeo-{env}-case-study-COSTING-RESPONSE-{version}"
+    description: >
+      This topic is used to REPLY Costing Requests and is targeted by the
+      `REPLY_TOPIC` header.
+      **You must grant PUBLISH access to our `svc-ccr-app` service account.**.
+      We use the
+      [**RecordNameStrategy**](https://docs.confluent.io/platform/current/schema-registry/serdes-develop/index.html#subject-name-strategy)
+      to infer the messages schema.
+      You have to define `key.subject.name.strategy` and
+      `value.subject.name.strategy` to
+      `io.confluent.kafka.serializers.subject.RecordNameStrategy` in your
+      consumer.
+      The schema below illustrates how Costing Response messages are
+      handled.
+           ![](https://user-images.githubusercontent.com/5501911/188920831-689cec5f-8dc3-460b-8794-0b54ec8b0ac8.png)
+    parameters:
+      env:
+        $ref: "#/components/parameters/Env"
+      version:
+        $ref: "#/components/parameters/Version"
+    bindings:
+      kafka:
+        groupId:
+          type: string
+          description: >
+            The groupId must be prefixed by your `svc` account, deliver by the
+            Adeo Kafka team.
+            This `svc` must have the read access to the topic.
+        key.subject.name.strategy:
+          type: string
+          description: >
+            We use the RecordNameStrategy to infer the messages schema.
+            Use
+            `key.subject.name.strategy=io.confluent.kafka.serializers.subject.RecordNameStrategy`
+            in your consumer configuration.
+        value.subject.name.strategy:
+          type: string
+          description: >
+            We use the RecordNameStrategy to infer the messages schema.
+            Use
+            `value.subject.name.strategy=io.confluent.kafka.serializers.subject.RecordNameStrategy`
+            in your consumer configuration.
+    messages:
+      costingResponse:
+        $ref: "#/components/messages/costingResponse"
+
+
+
+
+
+operations:
+    requestCosting:
+      action: receive
+      channel: 
+        $ref: '#/channels/costingRequest'
+      summary: |
+        [COSTING] Request one or more Costing calculation for any product
+      description: >
+        You can try a costing request using our [Conduktor producer
+        template](https://conduktor.url)
+      tags:
+        - name: costing
+      bindings:
+        kafka:
+          groupId:
+            type: string
+            description: >
+              The groupId must be prefixed by your `svc` account, deliver by the
+              Adeo Kafka team.
+              This `svc` must have the write access to the topic.
+          value.subject.name.strategy:
```

Comment:
as above... I don't think we have this in the bindings - are we planning to add it?